src/azure-cli/azure/cli/command_modules/synapse/tests/latest/test_synapse_scenario.py

```diff
@@ -1406,12 +1406,13 @@ def test_sql_pool(self):
         self.cmd('az synapse sql pool show --name {sql-pool} --workspace {workspace} --resource-group {rg}',
                  expect_failure=True)
 
-    @record_only()
+    @unittest.skip('Resource group \'rgtesting\' could not be found')
```

Comment:
Skip it temporarily as we cannot pass test lively as the test used fixed resources, will contact service team to fix it later.